<!DOCTYPE html>
    <html>
    <head>
        <meta charset="UTF-8">
        <title>SUMARY NUMBERS BC-0345797636</title>
        <style>
</style>
        
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
<style>
            body {
                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
                font-size: 14px;
                line-height: 1.6;
            }
        </style>
        <style>
.task-list-item { list-style-type: none; } .task-list-item-checkbox { margin-left: -20px; vertical-align: middle; }
</style>
        
        
        
    </head>
    <body class="vscode-body vscode-light">
        <h2 id="sumary-numbers-bc-0345797636">SUMARY NUMBERS BC-0345797636</h2>
<p><a href="../stats.html">Volver Anexos Stats</a></p>
<table>
<thead>
<tr>
<th>Parametro</th>
<th>Numero</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>raw total sequences</strong></td>
<td><strong>66723700</strong></td>
</tr>
<tr>
<td>filtered sequences</td>
<td>0</td>
</tr>
<tr>
<td>sequences</td>
<td>66723700</td>
</tr>
<tr>
<td>is sorted</td>
<td>1</td>
</tr>
<tr>
<td>1st fragments</td>
<td>33361850</td>
</tr>
<tr>
<td>last fragments</td>
<td>33361850</td>
</tr>
<tr>
<td><strong>reads mapped</strong></td>
<td><strong>2902</strong></td>
</tr>
<tr>
<td>reads mapped and paired</td>
<td>2878	# paired-end technology bit set + both mates mapped</td>
</tr>
<tr>
<td>reads unmapped</td>
<td>66720798</td>
</tr>
<tr>
<td>reads properly paired</td>
<td>2798	# proper-pair bit set</td>
</tr>
<tr>
<td>reads paired</td>
<td>66723700	# paired-end technology bit set</td>
</tr>
<tr>
<td>reads duplicated</td>
<td>0	# PCR or optical duplicate bit set</td>
</tr>
<tr>
<td>reads MQ0</td>
<td>0	# mapped and MQ=0</td>
</tr>
<tr>
<td>reads QC failed</td>
<td>0</td>
</tr>
<tr>
<td>non-primary alignments</td>
<td>0</td>
</tr>
<tr>
<td>total length</td>
<td>9574261798	# ignores clipping</td>
</tr>
<tr>
<td>total first fragment length</td>
<td>4789862929	# ignores clipping</td>
</tr>
<tr>
<td>total last fragment length</td>
<td>4784398869	# ignores clipping</td>
</tr>
<tr>
<td><strong>bases mapped</strong></td>
<td><strong>407785</strong>	# ignores clipping</td>
</tr>
<tr>
<td>bases mapped (cigar)</td>
<td>407785	# more accurate</td>
</tr>
<tr>
<td>bases trimmed</td>
<td>0</td>
</tr>
<tr>
<td>bases duplicated</td>
<td>0</td>
</tr>
<tr>
<td>mismatches</td>
<td>1160	# from NM fields</td>
</tr>
<tr>
<td>error rate</td>
<td>2.844636e-03	# mismatches / bases mapped (cigar)</td>
</tr>
<tr>
<td>average length</td>
<td>143</td>
</tr>
<tr>
<td>average first fragment length</td>
<td>144</td>
</tr>
<tr>
<td>average last fragment length</td>
<td>143</td>
</tr>
<tr>
<td>maximum length</td>
<td>151</td>
</tr>
<tr>
<td>maximum first fragment length</td>
<td>151</td>
</tr>
<tr>
<td>maximum last fragment length</td>
<td>151</td>
</tr>
<tr>
<td>average quality</td>
<td>36.0</td>
</tr>
<tr>
<td>insert size average</td>
<td>212.7</td>
</tr>
<tr>
<td>insert size standard deviation</td>
<td>100.5</td>
</tr>
<tr>
<td>inward oriented pairs</td>
<td>976</td>
</tr>
<tr>
<td>outward oriented pairs</td>
<td>463</td>
</tr>
<tr>
<td>pairs with other orientation</td>
<td>0</td>
</tr>
<tr>
<td>pairs on different chromosomes</td>
<td>0</td>
</tr>
<tr>
<td>percentage of properly paired reads (%)</td>
<td>0.0</td>
</tr>
</tbody>
</table>

    </body>
    </html>